src/app/core/common-components/entity-subrecord/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -283,6 +285,10 @@ export class EntitySubrecordComponent<T extends Entity> implements OnChanges {
     }
   }
 
+  setFilteredData( filterData: any) {
+    this.filteredData.emit(filterData);
+  }
```

Comment:
as a one-liner, we can inline this instead (i.e. remove this method and directly call filteredData.emit )

Reply:
or maybe instead we use this to map the filteredData so that it outputs an array of entities instead of the weird, wrapped objects that the datasource uses?

Reply:
@sleidig, I have removed the function and calling directly `filteredData.emit`